pytest_test_groups/__init__.py

```diff
@@ -1,13 +1,20 @@
-import math
+# -*- coding: utf-8 -*-
```

Comment:
I don't see why this is necessary here? It seems extraneous/

Reply:
Force of habit to always make sure to have the encoding header set. Need it removed?